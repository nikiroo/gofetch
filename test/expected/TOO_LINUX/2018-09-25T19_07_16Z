            CHROMEOS 69 : LES APPLICATIONS LINUX SONT LÀ           

  2018/09/25

  o News link: https://www.toolinux.com/?ChromeOS-69-les-applications-Linux-sont-la
  o Source link: 


    ChromeOS 69 : les applications Linux sont là
    
    mardi 25 septembre 2018
    
    C’était  un privilège réservé aux développeurs. Désormais, avec
    Chrome  OS  69,  tout  le  monde peut profiter des applications
    Linux sur Chromebook, mais pas encore sur tous les modèles.
    
    C’était  un  navigateur  amélioré.  C’est  désormais un système
    d’exploitation   capable  de  faire  tourner  des  applications
    progressives  du  web,  les  applications  Android et désormais
    aussi  certaines  applications  Linux  (normal  pour un système
    basé sur un noyau Linux, non ?).
    
    [1]ChromeOS  69  apporte  ,  l’air  de rien, un grand nombre de
    nouveautés  :  la  barre  Omnibox  améliorée, des emojis et une
    solution  pour  filtrer  la  lumière bleue. Autre arrivée : les
    commandes vocales.
    
    Quant  aux  applications Linux, elles sont réservées à certains
    appareils  : Acer (11, Spin 11, Chromebook 15), Asus Chromebook
    Flip,  Google  Pixelbook,  HP  Chromebook  x360  et  x2, Lenovo
    Thinkpad  11e  et  Samsung  Chromebook Plus (V1 et V2). Si vous
    souhaitez  installer  une  applications  Linux,  il  suffit  de
    disposer du paquet .deb correspondant.
    
    [2]
    
    
    
    [1] https://chromeunboxed.com/news/chrome-os-69-released-chrom-
    ebooks
    
    [2] https://www.toolinux.com/?ChromeOS-69-les-applications-Lin-
    ux-sont-la#forum



