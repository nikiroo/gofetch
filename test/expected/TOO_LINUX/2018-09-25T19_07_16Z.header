0ChromeOS 69 : les applications Linux sont là	null/TOO_LINUX/2018-09-25T19_07_16Z		70
i  2018/09/25
i
i    C’était  un privilège réservé aux développeurs. Désormais, avec
i    Chrome  OS  69,  tout  le  monde peut profiter des applications
i    Linux  sur  Chromebook,  mais  pas encore sur tous les modèles.
i    C’était  un  navigateur  amélioré.  C’est  désormais un système
i    d’exploitation   capable  de  faire  tourner  des  applications
i    progressives  du  web,  les  applications  Android et désormais
i    (...)
i
