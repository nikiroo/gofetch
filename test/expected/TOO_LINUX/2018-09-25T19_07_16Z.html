<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story'>
	<h1>ChromeOS 69 : les applications Linux sont là</h1>
	<div class='details'>(2018/09/25)</div>
	<br/>
    <ul>
        <li>News link: <a href='https://www.toolinux.com/?ChromeOS-69-les-applications-Linux-sont-la'>https://www.toolinux.com/?ChromeOS-69-les-applications-Linux-sont-la</a></li>
        <li>Source link: <a href=''></a></li>
    </ul>
	<br/>
	<div class='content' style='text-align: justify'>
		ChromeOS 69 : les applications Linux sont là<br/><br/>mardi 25 septembre 2018<br/><br/>C’était un privilège réservé aux développeurs. Désormais, avec Chrome OS 69, tout le monde peut profiter des applications Linux sur Chromebook, mais pas encore sur tous les modèles.<br/><br/>C’était un navigateur amélioré. C’est désormais un système d’exploitation capable de faire tourner des applications progressives du web, les applications Android et désormais aussi certaines applications Linux (normal pour un système basé sur un noyau Linux, non ?).<br/><br/>[1]ChromeOS 69 apporte , l’air de rien, un grand nombre de nouveautés : la barre Omnibox améliorée, des emojis et une solution pour filtrer la lumière bleue. Autre arrivée : les commandes vocales.<br/><br/>Quant aux applications Linux, elles sont réservées à certains appareils : Acer (11, Spin 11, Chromebook 15), Asus Chromebook Flip, Google Pixelbook, HP Chromebook x360 et x2, Lenovo Thinkpad 11e et Samsung Chromebook Plus (V1 et V2). Si vous souhaitez installer une applications Linux, il suffit de disposer du paquet .deb correspondant.<br/><br/>[2]<br/><br/><br/><br/>[1] https://chromeunboxed.com/news/chrome-os-69-released-chromebooks<br/><br/>[2] https://www.toolinux.com/?ChromeOS-69-les-applications-Linux-sont-la#forum
	</div>
<hr/>
</div>
</body>
